app/src/androidTest/java/com/github/geohunt/app/LeaderboardTest.kt

```diff
@@ -42,10 +43,15 @@ class LeaderboardTest {
         return object : User {
             override var displayName: String? = names[pos]
             override val uid: String = pos.toString()
-            override val profilePicture: LazyRef<Bitmap> = MockLazyRef("1") { throw NotImplementedError() }
+            override val profilePicture: LazyRef<Bitmap> = object : LazyRef<Bitmap> {
+                override val id: String = "0"
+                override var value: Bitmap? = Bitmap.createBitmap(100, 100, Bitmap.Config.ARGB_8888)
+
+                override fun fetch(): Task<Bitmap> = Tasks.forResult(value)
+            }
             override val challenges: List<LazyRef<Challenge>> = listOf()
             override val hunts: List<LazyRef<Challenge>> = listOf()
-            override var score: Number = 1500 - pos * 100
+            override var score: Double = 1500.0 - pos * 100
```

Comment:
Consider rewriting the code to: override var score: Double = 1500.0 - (pos * 100). It would improve the readability of the code.